isis/tests/CubeStretchTest.cpp

```diff
@@ -37,6 +38,39 @@ TEST(CubeStretch, CopyConstructor) {
   EXPECT_EQ(cubeStretch.getBandNumber(), copyStretch.getBandNumber());
 }
 
+TEST(CubeStretch, BlobConstructor) {
+  // Set Stretch
+  Isis::CubeStretch cubeStretch("TestStretch", "testType", 2);
+  Isis::CubeStretch cubeStretchFromBlob(cubeStretch.toBlob());
+
+
+  EXPECT_STREQ(cubeStretchFromBlob.getName().toLatin1().data(), cubeStretch.getName().toLatin1().data());
+  EXPECT_STREQ(cubeStretchFromBlob.getType().toLatin1().data(), cubeStretch.getType().toLatin1().data());
+
+  EXPECT_STREQ(cubeStretchFromBlob.getType().toLatin1().data(), cubeStretch.getType().toLatin1().data());
+  EXPECT_EQ(cubeStretchFromBlob.getBandNumber(), cubeStretch.getBandNumber());
+};
+
+TEST_F(DefaultCube, CubeStretchWriteRead) {
```

Comment:
Use SmallCube here, it has much less overhead in its initialization. It also has multiple bands so we could test writing and reading multiple stretches for different bands.